utils/check_dashboard_name_uniqueness.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+const glob = require('glob');
+const {
+  readQuickstartFile,
+  removeRepoPathPrefix,
+  findMainQuickstartConfigFiles,
+} = require('./helpers');
+
+const getQuickstartDashboardConfigs = (quickstartConfigPath) => {
+  const splitConfigPath = quickstartConfigPath.split('/');
+  splitConfigPath.pop();
+  const globPattern = `${splitConfigPath.join('/')}/dashboards/*.+(json)`;
+
+  return glob.sync(globPattern);
+};
+
+const getMatchingNames = (namesAndPaths) => {
+  return namesAndPaths.reduce((acc, { name, path }) => {
+    const duplicates = namesAndPaths.filter(
+      (quickstart) => quickstart.name === name && quickstart.path !== path
+    );
+
+    return [...new Set([...acc, ...duplicates])];
+  }, []);
+};
+
+const cleanQuickstartName = (str) =>
+  str
+    .trim()
+    .toLowerCase()
+    .replace(/\s+/g, '-')
+    .replace(/-+/, '-')
+    .replace(/[^a-z0-9-]/g, '');
+
+const findMatchingDashboardNames = () => {
```

Comment:
could you add the spec for this function, like there is in other files?

https://github.com/tyreer/newrelic-quickstarts/blob/03af1a4960bd4364d9c728517dd45c315c455791/utils/check_quickstart_uniqueness.js#L36-L40